bin/serve.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http_server/http_server.dart' show VirtualDirectory;
+import 'package:path/path.dart';
+
+final webroot = join(dirname(dirname(Platform.script.toFilePath())), 'build');
+
+Future<void> main() async {
+  final virDir = new VirtualDirectory(webroot);
+
+  // Set up a directory handler to serve index.html files.
+  virDir.allowDirectoryListing = true;
+  virDir.directoryHandler = (dir, request) {
+    final indexUri = new Uri.file(dir.path).resolve('index.html');
+    virDir.serveFile(new File(indexUri.toFilePath()), request);
+  };
+
+  // TODO(dantup): How to decide port?
+  final server = await HttpServer.bind('127.0.0.1', 8765);
```

Comment:
I would suggest defaulting to an easy to remember number like 8080 and accepting an alternate port as a command line argument.

Reply:
I was hoping to figure out how Flutter/Dart VM come up with a random port, so if it's taken we don't have to fail. Maybe they just start at a number and keep increasing until they successfully bind?

Reply:
I think we should come up on a specific port - either the default one, or one specified by the user (having a random high number port is less ergonomic).